packages/analytics/src/helpers.ts

```diff
@@ -255,6 +256,13 @@ function wrapGtag(
       } else if (command === GtagCommand.CONSENT) {
         // If CONFIG, second arg must be measurementId.
         gtagCore(GtagCommand.CONSENT, 'update', gtagParams as ConsentSettings);
+      } else if (command === GtagCommand.GET) {
```

Comment:
I think we talked about giving `SET` its own case and then making the final `else` fallback block just pass everything straight through in order to not only cover `GET` but anything else we haven't thought of.